src/api/index.ts

```diff
@@ -766,6 +772,62 @@ class API {
 
   /**
    * Gets realm & client roles that are mapped to a particular realm group.
```

Comment:
i think git's rebase inserted your code in a line or two too low. looks like it jumped between an existing method & its comment